TACC/Core-Portal
# task/FP-1939 cleanup searchbar and sidebar components

## Overview

The goal was to cleanup any unused code, components and modules after the implementation of [common searchbar](https://github.com/TACC/Core-Portal/pull/724) and [common sidebar](https://github.com/TACC/Core-Portal/pull/731) components.

## Related

* [FP-1939](https://jira.tacc.utexas.edu/browse/FP-1939)

## Changes

- Mostly deleting unused components and deleting unused lines of css 
- Removed some previously commented out code in the Section component

## Testing

1. Go to the Portal 
2. Test out general functionality like navigating between pages and searching in the Data Files page and Site Search page
3. Ensure everything works properly and there are no UI inconsistencies 

## Notes

One point for future discussion should be the commented out lines of code that were removed from the [Section](https://github.com/TACC/Core-Portal/blob/main/client/src/components/_common/Section/Section.jsx#L114) component. @wesleyboar had written this code and the plan was to have `sidebar` props in the Section component which would be used to pass in the sidebar component to be used in that Section. This approach is similar to what is done with the [header](https://github.com/TACC/Core-Portal/blob/main/client/src/components/_common/Section/Section.jsx#L108#L110) and [content](https://github.com/TACC/Core-Portal/blob/main/client/src/components/_common/Section/Section.jsx#L104#L107) as those are separate props whose values are passed into the Section. Currently the sidebar component is sent in the `content` prop. The functionality works fine at the moment but we can have a discussion if a better design would be to have explicit sidebar props or any other alternatives the team suggests.
